algorithms/sort/radix_sort.py

```diff
@@ -28,7 +25,7 @@ def radix_sort(arr, simulation=False):
 
         if simulation:
             iteration = iteration + 1
-            print("iteration",iteration,":",*arr)
+            print("iteration", iteration, ":", *arr)
 
         position *= 10
-    return arr
+    return arr
```

Comment:
Nit: could you add a new line at the end?

Reply:
OK